components/global-navigation-bar/index.jsx

```diff
@@ -96,10 +84,7 @@ GlobalNavigationBar.propTypes = {
 	theme: PropTypes.oneOf(['light', 'dark']),
 };
 
-GlobalNavigationBar.defaultProps = {
-	cloud: 'default',
-	theme: 'dark',
-};
```

Comment:
Why did you remove these?  These are convenient if a developer has to write less code.

Reply:
For `cloud`, the default appearance is assumed if nothing is set. The SLDS class `.slds-context-bar_theme-default` does not exist, but `.slds-context-bar_theme-marketing` does. Right now the `cloud` PropType is `string`, but perhaps it should be more specific such as `cloud: PropTypes.oneOf(['marketing'])`?

For `theme`, I'm actually not sure if this is still needed after re-review. At the very least I know the default is to not have any theme class applied, which is why I removed it here... but looking through the SLDS classes, I don't see any `.slds-context-bar_theme-light` or `.slds-context-bar_theme-dark` definitions... on the documentation site there's only one reference to those here:
<img width="981" alt="screen shot 2019-03-07 at 3 53 17 pm" src="https://user-images.githubusercontent.com/991513/53995097-40cd4080-40f1-11e9-85ed-7df4cf197907.png">

Perhaps we should get rid of theme altogether? @interactivellama, do you happen to know anything else about the light / dark themes?

Reply:
I'm not seeing any reference to light and dark themes in the CSS either.